tests/sdk/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"boscoin.io/sebak/lib/client"
+	"fmt"
+	"net/http"
+	"os"
+)
+
+func main() {
+	c := client.NewClient("https://127.0.0.1:2830")
+	headers := http.Header{}
+	headers.Set("Content-Type", "application/json")
+	a, e := c.LoadAccount("GDIRF4UWPACXPPI4GW7CMTACTCNDIKJEHZK44RITZB4TD3YUM6CCVNGJ")
+	if e != nil {
+		fmt.Println(e)
+	}
+	fmt.Println(a)
+
+	os.Exit(0)
```

Comment:
That's unnecessary

Reply:
As I mentioned before, It is a dummy code.
The main.go and relevant code will be added to test client and relevant node features

Reply:
I am just referring to the `os.Exit` statement

Reply:
I know what the os.Exit(0) means and it is redundant.
It is a `dummy script` to make sure that always exit with 0 returned value.